remoteappmanager/handlers/admin/admin_home_handler.py

```diff
@@ -9,4 +9,4 @@ class AdminHomeHandler(BaseHandler):
     @web.authenticated
     @gen.coroutine
     def get(self):
-        self.render('admin/page.html')
```

Comment:
I changed the name of the admin and user page files because when doing `{% extends "page.html" %}` they were loading themselves resulting on an infinite loop.